### PR TITLE
removed ModuleDict, since we only have one metric and removed argmax …

### PR DESCRIPTION
**Pull request by RSM-git**
…in training and validation steps as the metric performs it by itself, when computing

### CODE REVIEW

File: src/models/model.py [hunk 1]
```diff
@@ -100,14 +87,16 @@ def training_step(self, batch: dict, batch_idx: int) -> torch.Tensor:
         input_ids = batch["input_ids"]
         attention_mask = batch["attention_mask"]
         labels = batch["label"]
-        logits = self(input_ids, attention_mask)
-        loss = self.criterion(logits, labels)
-        predictions = logits.argmax(dim=1)
-        self.training_metrics["accuracy"](predictions, labels)
+
+        predictions = self(input_ids, attention_mask)
```
Comment (Tolsager) [inline src/models/model.py]:
```suggestion
        predictions = self.model(input_ids, attention_mask).logits
```

File: src/models/model.py [hunk 2]
```diff
@@ -127,14 +116,16 @@ def validation_step(self, batch: dict, batch_idx: int) -> torch.Tensor:
         input_ids = batch["input_ids"]
         attention_mask = batch["attention_mask"]
         labels = batch["label"]
-        logits = self(input_ids, attention_mask)
-        loss = self.criterion(logits, labels)
-        predictions = logits.argmax(dim=1)
-        self.validation_metrics["accuracy"](predictions, labels)
+
+        predictions = self(input_ids, attention_mask)
```
Comment (Tolsager) [inline src/models/model.py]:
```suggestion
        predictions = self.model(input_ids, attention_mask).logits
```

Comment (RSM-git) [reply src/models/model.py]: Nogen fordele ved at kalde self.model, når forward returner logits alligevel?

Comment (Tolsager) [reply src/models/model.py]: det er en lightning standard. De anbefaler, at man kun bruger forward til inference, men kan ikke huske hvorfor

**Review by Tolsager — APPROVED** [review]
